pkg/apis/authentication/types.go

```diff
@@ -135,7 +135,9 @@ type TokenRequestSpec struct {
 	ExpirationSeconds int64
 
 	// BoundObjectRef is a reference to an object that the token will be bound to.
-	// The token will only be valid for as long as the bound objet exists.
+	// The token will only be valid for as long as the bound object exists.
+	// NOTE: The API server will validate the BoundObjectRef, but other audiences
+	// may not. Keep ExpirationSeconds small if you want prompt revocation.
```

Comment:
If you want this to make it into api docs, you need to make the same changes in:

staging/src/k8s.io/api/authentication/v1
staging/src/k8s.io/api/authentication/v1beta1

Then run make update.

Reply:
Done.